src/client/Microsoft.Identity.Client/Internal/Requests/Silent/CacheSilentStrategy.cs

```diff
@@ -161,7 +164,8 @@ private async Task<AuthenticationResult> CreateAuthenticationResultAsync(MsalAcc
                 AuthenticationRequestParameters.AuthenticationScheme,
                 AuthenticationRequestParameters.RequestContext.CorrelationId,
                 TokenSource.Cache,
-                AuthenticationRequestParameters.RequestContext.ApiEvent);
+                AuthenticationRequestParameters.RequestContext.ApiEvent,
```

Comment:
For testing I suggest you look at unit tests which start from an existing cache file, such as `MultiTenantTokenCache.json`